src/components/AppPagination.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="text-center">
```

Comment:
centering of the pagination should be on the page level, not on the component level

Reply:
this block and css property were together with vuetify pagination. I thought that I can't remove it.